cloudify_cli/execution_events_fetcher.py

```diff
@@ -101,7 +131,10 @@ def wait_for_execution(client,
     events_fetcher = ExecutionEventsFetcher(client, execution.id,
                                             include_logs=include_logs)
 
-    # Poll for execution status until execution ends
+    # Poll for execution status and execution logs, until execution ends
+    # and we receive an event of type in WORKFLOW_END_TYPES
+    execution_ended = False
+    events_watcher = EventsWatcher(events_handler)
     while True:
```

Comment:
Yeah, you'r right. I've added a comment to the gist, though again, you are right, it might not be as cleaner as I've hoped originally...